drupal8-apache/7.0/usr/local/share/drupal8/install_finalise.sh

```diff
@@ -45,11 +34,11 @@ set +e
 is_nfs
 IS_NFS=$?
 set -e
-if [ "$IS_NFS" -ne 0 ]; then
-  chown -R www-data:www-data "$SETTINGS_DIR/files/"
+if [ "$IS_NFS" -ne 0 ] && [ -d "$SETTINGS_DIR/files/" ]; then
+  chown -R "$APP_USER:$APP_GROUP" "$SETTINGS_DIR/files/"
 fi
 
-as_build "drush cache-rebuild" "/app/docroot"
+as_code_owner "drush cache-rebuild" "/app/docroot"
```

Comment:
This needs to come out. We're not installing the site here any more, so a cache rebuild causes the images to stop building.